Yoast/wordpress-seo
# Replaced banner image with latest version

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Test instructions

This PR can be tested by following these steps:

* See original issue for the banner images. 
* Under SEO check the banners on the right side: refresh a couple of times to see that the old one is replaced by the new one 

## Quality assurance


Fixes #9444
